EndlessClient/Dialogs/GuildDialog.cs

```diff
@@ -213,9 +244,36 @@ private void SetState(GuildDialogState state)
                         AddItemToList(disbandItem, sortList: false);
                     }
                     break;
+                case GuildDialogState.Modify:
+                    {
+                        ListItemType = ListDialogItem.ListItemStyle.Large;
```

Comment:
Should this be changing the list item style to `Small`? All the list dialog items below are `ListItemStyle.Small`.

Also consider that changing the list item style within the dialog necessitates changing it *back*. @AstralSkies used a stack of dialog states in his PR, I think maybe it'd be worth doing the following:
1. Track a stack of DialogState/DialogStyle
2. Each call to `SetState` pushes the current state/style onto the stack
3. Each call to `BackAction` pops the previous state/style off the stack and sets to that state.

This way we don't have to maintain this complex web of which state goes to which style etc. etc. We just use what's on the stack.

Reply:
https://github.com/AstralSkies/EndlessClient/blob/guild_testing/EndlessClient/Dialogs/GuildDialog.cs

Yeah my branch uses a stack to manage dialog states. I've linked it

Reply:
Now tracking state/buttons/listitemtype per state and using stack

Reply:
Changed style to `Small`